etc/adminhtml/system.xml

```diff
@@ -42,7 +42,7 @@
                     <label>Enable Credit Card Solution</label>
                     <comment>Powerful payment features that allows you to easily and securely accept credit/debit card payments on your store.</comment>
                     <field id="active" translate="label comment" type="select" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
-                        <label>Enable/Disable</label>
+                        <label>Enabled/Disabled</label>
```

Comment:
This should say `Enabled?` as I suggested when I previously mentioned it. This still doesn't make sense

Reply:
@jonrandy why as English native speaker you just won't simply say what should be here? Whatever you say, I will add it here.

Reply:
I have said. Twice.

`Enabled?`

Reply:
sorry.. I misunderstood. 
I changed it.